apps/system/js/advanced_telemetry.js

```diff
@@ -361,12 +357,17 @@
       function onload() {
         loginfo('Transmitted Successfully.');
         AdvancedTelemetry.prototype.clearPayload();
+        // Start a new batch.
+        self.metrics = new BatchTiming(false, false);
```

Comment:
Code maintainability/readability comment: In my opinion, booleans as function arguments diminishes code readability. Alternatives include using constants (e.g., STARTUP/EXISTING_SESSION, RETRY/NORMAL) instead of booleans. Also, in this case, I could see having a wrapper functions for BatchTiming such as RetryBatchTiming, StartupBatchTiming, etc. to reduce the use of  booleans.